preupg/xml_to_json.py

```diff
@@ -0,0 +1,404 @@
+# -*- coding: utf-8 -*-
+
+import re
+import json
+import logging
+import xml.etree.ElementTree as ET
+
+from preupg.xmlgen.xml_tags import GLOBAL_DIC_VALUES
+from preupg.logger import log_message
+from preupg.exception import (MissingElementException,
+                              AttrNotFoundException,
+                              EmptyTextException)
+
+
+class XmlJsonConverter(object):
+    '''
+        Converts xml file (result.xml) into more readable
+        JSON (result.json) file
+        By default converts:
+            from <TestResult>:
+                end-time(attribute), start-time(attribute)
+                inside:
+                    target-address(text)
+            from <rule-result>:
+                inside:
+                    result(text), set-value(idref), check-import(text)
+    '''
+    def __init__(self, file_name, additional_rule_elements=None, verbose=False):
```

Comment:
Please make use of the verbose parameter. It would be great if you could add a --verbose command line option for the user.